### ISSUE TITLE
Tutorial on website does not work as specified

### ISSUE THREAD

**Issue by kzollman** [openmole/openmole#393]
Hello. I'm just starting to learn about OpenMole, and thank you for your hard work on the project.

I followed the tutorial on the website at https://openmole.org/Step+By+Step+Introduction.html  results in an error message (reproduced here).  I followed the tutorial exactly, including downloading the linked ants.nlogo, rather than using a local copy.  Compilation failed.  I'm guessing the link to the ants.nlogo file is to a file that is out of date.  (Perhaps this would be as simple as updating the website with a more recent Ants.nlogo.) When I load Ants.nlogo from the most recent version of NetLogo I have on my computer it compiles correctly.  

Here is the error when I follow the tutorial:

```
org.openmole.core.exception.InternalProcessingError: Error in antsTask@7218261 for context values {diffusionRate=21.0, evaporationRate=9.0, openmole$seed=-1422443212459168921, population=125.0, seed=0}
	at org.openmole.core.workflow.tools.InputOutputCheck$.$anonfun$perform$1(InputOutputCheck.scala:91)
	at org.openmole.core.expansion.FromContext$$anon$9.from(FromContext.scala:163)
	at org.openmole.core.workflow.task.Task.perform(Task.scala:68)
	at org.openmole.core.workflow.task.Task.perform$(Task.scala:66)
	at org.openmole.plugin.task.netlogo6.NetLogo6Task.perform(NetLogo6Task.scala:110)
	at org.openmole.core.workflow.job.MoleJob.perform(MoleJob.scala:102)
	at org.openmole.core.workflow.execution.local.LocalExecutor.$anonfun$run$2(LocalExecutor.scala:63)
	at org.openmole.core.workflow.execution.local.LocalExecutor.$anonfun$run$2$adapted(LocalExecutor.scala:61)
	at scala.collection.Iterator.foreach(Iterator.scala:944)
	at scala.collection.Iterator.foreach$(Iterator.scala:944)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1432)
	at scala.collection.IterableLike.foreach(IterableLike.scala:71)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at org.openmole.core.workflow.execution.local.LocalExecutor.$anonfun$run$1(LocalExecutor.scala:61)
	at org.openmole.core.outputmanager.OutputManager$.withStreamOutputs(OutputManager.scala:123)
	at org.openmole.core.workflow.execution.local.LocalExecutor.withRedirectedOutput(LocalExecutor.scala:116)
	at org.openmole.core.workflow.execution.local.LocalExecutor.run(LocalExecutor.scala:58)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.openmole.core.exception.UserBadDataError: Error while opening the file /home/kzollman/.openmole/hamilton/.tmp/ba6b5a87-884a-4255-8c74-7f1b12a224d3/execution4429a61d-0c79-417d-92ed-0fa9d2b61ea4/netlogoworkpsace9fcb89f2-f62e-4ff9-b2bd-0c13c3c8f467/projects/ants.nlogo:
 | java.lang.IllegalStateException: unknown NetLogo version: NetLogo 5.0.4
 | 	at org.nlogo.headless.HeadlessModelOpener.openFromModel(HeadlessModelOpener.scala:31)
 | 	at org.nlogo.headless.HeadlessWorkspace.openModel(HeadlessWorkspace.scala:535)
 | 	at org.nlogo.headless.HeadlessWorkspace.open(HeadlessWorkspace.scala:502)
 | 	at org.openmole.plugin.tool.netlogo6.NetLogo6.open(NetLogo6.java:50)
 | 	at org.openmole.plugin.task.netlogo.NetLogoTask$.$anonfun$openNetLogoWorkspace$3(NetLogoTask.scala:79)
 | 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
 | 	at org.openmole.plugin.task.netlogo.NetLogoTask$.wrapError(NetLogoTask.scala:49)
 | 	at org.openmole.plugin.task.netlogo.NetLogoTask$.$anonfun$openNetLogoWorkspace$2(NetLogoTask.scala:79)
 | 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
 | 	at org.openmole.tool.thread.package$.withThreadClassLoader(package.scala:53)
 | 	at org.openmole.core.code.CodePackage.withThreadClassLoader(CodePackage.scala:49)
 | 	at org.openmole.core.code.CodePackage.withThreadClassLoader$(CodePackage.scala:48)
 | 	at org.openmole.core.workflow.dsl$.withThreadClassLoader(package.scala:46)
 | 	at org.openmole.plugin.task.netlogo.NetLogoTask$.openNetLogoWorkspace(NetLogoTask.scala:78)
 | 	at org.openmole.plugin.task.netlogo.NetLogoTask$.createInstance$1(NetLogoTask.scala:117)
 | 	at org.openmole.plugin.task.netlogo.NetLogoTask$.$anonfun$createPool$1(NetLogoTask.scala:126)
 | 	at org.openmole.tool.cache.WithNewInstance.apply(Pool.scala:43)
 | 	at org.openmole.plugin.task.netlogo.NetLogoTask.$anonfun$process$1(NetLogoTask.scala:286)
 | 	at org.openmole.core.expansion.FromContext$$anon$8.from(FromContext.scala:157)
 | 	at org.openmole.core.workflow.tools.InputOutputCheck$.$anonfun$perform$1(InputOutputCheck.scala:88)
 | 	at org.openmole.core.expansion.FromContext$$anon$9.from(FromContext.scala:163)
 | 	at org.openmole.core.workflow.task.Task.perform(Task.scala:68)
 | 	at org.openmole.core.workflow.task.Task.perform$(Task.scala:66)
 | 	at org.openmole.plugin.task.netlogo6.NetLogo6Task.perform(NetLogo6Task.scala:110)
 | 	at org.openmole.core.workflow.job.MoleJob.perform(MoleJob.scala:102)
 | 	at org.openmole.core.workflow.execution.local.LocalExecutor.$anonfun$run$2(LocalExecutor.scala:63)
 | 	at org.openmole.core.workflow.execution.local.LocalExecutor.$anonfun$run$2$adapted(LocalExecutor.scala:61)
 | 	at scala.collection.Iterator.foreach(Iterator.scala:944)
 | 	at scala.collection.Iterator.foreach$(Iterator.scala:944)
 | 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1432)
 | 	at scala.collection.IterableLike.foreach(IterableLike.scala:71)
 | 	at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
 | 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
 | 	at org.openmole.core.workflow.execution.local.LocalExecutor.$anonfun$run$1(LocalExecutor.scala:61)
 | 	at org.openmole.core.outputmanager.OutputManager$.withStreamOutputs(OutputManager.scala:123)
 | 	at org.openmole.core.workflow.execution.local.LocalExecutor.withRedirectedOutput(LocalExecutor.scala:116)
 | 	at org.openmole.core.workflow.execution.local.LocalExecutor.run(LocalExecutor.scala:58)
 | 	at java.lang.Thread.run(Thread.java:748)
	at org.openmole.plugin.task.netlogo.NetLogoTask$.wrapError(NetLogoTask.scala:53)
	at org.openmole.plugin.task.netlogo.NetLogoTask$.$anonfun$openNetLogoWorkspace$2(NetLogoTask.scala:79)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at org.openmole.tool.thread.package$.withThreadClassLoader(package.scala:53)
	at org.openmole.core.code.CodePackage.withThreadClassLoader(CodePackage.scala:49)
	at org.openmole.core.code.CodePackage.withThreadClassLoader$(CodePackage.scala:48)
	at org.openmole.core.workflow.dsl$.withThreadClassLoader(package.scala:46)
	at org.openmole.plugin.task.netlogo.NetLogoTask$.openNetLogoWorkspace(NetLogoTask.scala:78)
	at org.openmole.plugin.task.netlogo.NetLogoTask$.createInstance$1(NetLogoTask.scala:117)
	at org.openmole.plugin.task.netlogo.NetLogoTask$.$anonfun$createPool$1(NetLogoTask.scala:126)
	at org.openmole.tool.cache.WithNewInstance.apply(Pool.scala:43)
	at org.openmole.plugin.task.netlogo.NetLogoTask.$anonfun$process$1(NetLogoTask.scala:286)
	at org.openmole.core.expansion.FromContext$$anon$8.from(FromContext.scala:157)
	at org.openmole.core.workflow.tools.InputOutputCheck$.$anonfun$perform$1(InputOutputCheck.scala:88)
	... 18 more

```

**Comment by romainreuillon** [comment]
Nice catch, thanks for the report.

The version of Ants distributed in the doc is based on netlogo 5. I'll update to netlogo 6. In the meantime you can use a in you're script NetLogo5Task instead of a NetLogo6Task. It should do the trick.

**Comment by romainreuillon** [comment]
It has been fixed in fe4c71c0bb7f63b32976657aa532872b1f12635d

If you want the updated version of ant.netlogo, you may get it here: https://raw.githubusercontent.com/openmole/openmole/9-dev/openmole/bin/org.openmole.site/jvm/src/main/resources/script/ants.nlogo
